src/main/webapp/script.js

```diff
@@ -95,3 +95,14 @@ function createListElement(item) {
   liElement.innerHTML = '<a href=\"' + item.profilePageUrl + '\">' + item.username + '</a>';
   return liElement;
 }
+
+// Show fail message if appropriate.
+function signInFailMessage() {
```

Comment:
Maybe rename to showFailMessage(), it better reflects what this method actually does

Reply:
nevermind :')